pkg/api/login.go

```diff
@@ -278,7 +278,7 @@ func (hs *HTTPServer) loginUserWithUser(user *models.User, c *models.ReqContext)
 }
 
 func (hs *HTTPServer) Logout(c *models.ReqContext) {
-	if hs.Cfg.SAMLEnabled && hs.Cfg.SAMLSingleLogoutEnabled && hs.License.HasValidLicense() {
```

Comment:
As soon as we start to read these fields from the new settings provider,  we'll be able to remove them. However, I didn't do it yet because they're still being used by some tests, which still need to be updated / fixes (see PR's description).

Reply:
It's only being used from GE code, so I finally decided to remove it on this PR.